Purchases/Misc/Obsoletions.swift

```diff
@@ -296,11 +296,11 @@ public extension Purchases {
      * If the purchase was not successful, there will be an `NSError`.
      * If the user cancelled, `userCancelled` will be `YES`.
      */
-    @available(iOS, introduced: 12.2, obsoleted: 12.2, renamed: "purchase(product:discount:completion:)")
-    @available(tvOS, introduced: 12.2, obsoleted: 12.2, renamed: "purchase(product:discount:completion:)")
-    @available(watchOS, introduced: 6.2, obsoleted: 6.2, renamed: "purchase(product:discount:completion:)")
-    @available(macOS, introduced: 10.14.4, obsoleted: 10.14.4, renamed: "purchase(product:discount:completion:)")
-    @available(macCatalyst, introduced: 13.0, obsoleted: 13.0, renamed: "purchase(product:discount:completion:)")
+    @available(iOS, introduced: 12.2, unavailable, renamed: "purchase(product:discount:completion:)")
+    @available(tvOS, introduced: 12.2, unavailable, renamed: "purchase(product:discount:completion:)")
+    @available(watchOS, introduced: 6.2, unavailable, renamed: "purchase(product:discount:completion:)")
+    @available(macOS, introduced: 10.14, unavailable, renamed: "purchase(product:discount:completion:)")
```

Comment:
was the change from `10.14.4` -> `10.14` intentional?

Reply:
it should still be `10.14.4` I think, [based on the docs](https://developer.apple.com/documentation/storekit/skproductdiscount/3043535-identifier)

Reply:
good to go after that though!

Reply:
Thanks great catch!! I think I messed it up when I did multi-column editing. That's probably what the test failure is.

Reply:
Fixed.